docs/source/links/state.md

```diff
@@ -0,0 +1,609 @@
+---
+title: apollo-link-state
+description: Manage your local data with Apollo Client
+---
+
+Managing remote data from an external API is simple with Apollo Client, but
+where do we put all of our data that doesn't fit in that category? Nearly all
+apps need to centralize client-side data from user interactions and device APIs
+somewhere!
+
+In the past, Apollo users stored their application's local data in a separate
+Redux or MobX store. With `apollo-link-state`, you no longer have to maintain a
+second store for local state. Your Apollo Client cache is your single source of
+truth that holds all of your local data alongside your remote data. To access or
+update your local state, you use GraphQL queries & mutations just like you would
+for data from a server.
+
+When you use Apollo Client to manage your local state, you get all of the same
+benefits you know and love like caching and offline persistence without having
+to set these features up yourself. 🎉 On top of that, you also benefit from the
+excellent developer experience of [Apollo
+DevTools](https://github.com/apollographql/apollo-client-devtools) for painless
```

Comment:
I think it's actually worth having an example somewhere with a screenshot of using `@client` queries from the devtools! It's so cool!  It might be worth mentioning in that section that schema introspection (ie autocomplete) doesn't work on `@client` queries though (w/ link to an issue, if we think that in an ideal world that would work).

Reply:
I love that idea!! I think mentioning the lack of schema introspection would fit well at the bottom section where I talk about typechecking.